hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/action/TestCleanPlanner.java

```diff
@@ -122,10 +132,48 @@ void testGetDeletePaths(HoodieWriteConfig config, String earliestInstant, List<H
     assertEquals(expected, actual);
   }
 
+  @ParameterizedTest
+  @MethodSource("incrCleaningPartitionsTestCases")
+  void testPartitionsForIncrCleaning(HoodieWriteConfig config, String earliestInstant,
```

Comment:
can we also cover a non-partitioned table?

Reply:
sure.